Elering: add all Nordpool regions

lka-hh opened the pull request:
additional regions for norway (1-5) and sweden (1-4)  in line 14, validvalues

lka-hh committed:
Update elering.yaml

additional regions for norway (1-5) and sweden (1-4) line 14, validvalues

lka-hh committed:
Merge branch 'master' into patch-1

andig commented:
@lka-hh this invalidates the description?

/cc @naltatis how to you want to handle regions?

andig commented on templates/definition/tariff/elering.yaml:
```diff
@@ -11,7 +11,7 @@ group: price
   - name: region
     example: ee
-    validvalues: ["ee", "lt", "lv", "fi"]
+    validvalues: ["ee", "lt", "lv", "fi","NO1","NO2","NO3","NO4","NO5","SE1","SE2","SE3","SE4"]
```
Can we please use the same casing and spacing as for the others?

naltatis commented:
> how to you want to handle regions?

@andig yes, we should introduce a field with structured country data to improve the documentation. Currently we (mis)use the descriptions field for this. When we have it structured we could introduce proper sections on the documentation page and later build multilevel selection for tariffs in config ui: "select you country > select a tariff for your country".
For now lets stick to the description workaround.

@lka-hh can you update the description accordingly?

lka-hh committed:
Update elering.yaml

fixed spacing and casing

lka-hh committed:
Update elering.yaml

andig commented:
@lka-hh out of curiosity: shouldn't the same be possible for Energinet? They have dk1,2 however and those are not present here?

andig commented:
ping @lka-hh

lka-hh commented:
Hej, 
mal sehen ob ich das richtig verstehe: 
- Wir wollen alle Regionen von Nordpool in ein template bringen
- wir wollen die Beschreibung ändern so dass deutlich wird was alles vom Template erfasst ist .

Ich wårde das Template Nordpool taufen, die Beschreibung ändern/ergänzen und alle auf der Nordpool-homepage vorhandenen Regionen eintragen. 

Ich bin neu hier und haben keine Erfahrungen in der Handhabung von github und dessen Funktionen. Bin neugierig und will gerne was neues lernen. Brauche aber vermutlich eine Weile wenn ich es mir selbst beibringe. Vielleicht könnt ihr mir tips geben und habt einen Vorschlag wo ich mehr dazu lesen kann, wie man hier am besten arbeitet

lka-hh committed:
Update elering.yaml

changed template name to Nordpool, 
regions for Austria, Belgium, Denmark, France, Poland, Romania added in line 14

andig commented on templates/definition/tariff/elering.yaml:
```diff
@@ -1,19 +1,19 @@
   - preset: tariff-base
 render: |
-  type: elering
+  type: Nordpool
```
```suggestion
  type: elering
```

there is no "Nordpool" Go code ;)

RTTTC replied:
This looks strange to me, as Elering only provides data for the 4 countries (FI, EE, LV, LT) 
???

RTTTC commented:
This looks strange to me, as Elering only provides data for the 4 countries (FI, EE, LV, LT)
???

andig commented:
Do we still need Elering at all after https://github.com/evcc-io/evcc/pull/17628?
